www/front_src/src/Resources/Details/Body/tabs/Details/index.tsx

```diff
@@ -135,39 +140,31 @@ const DetailsTab = ({ details }: Props): JSX.Element => {
         )}
       </Grid>
       {details.performance_data && (
-        <Grid item>
-          <ExpandableCard
-            title={labelPerformanceData}
-            content={details.performance_data}
-          />
-        </Grid>
+        <ExpandableCard
+          title={labelPerformanceData}
+          content={details.performance_data}
+        />
       )}
       {details.command_line && (
-        <Grid item>
-          <Card>
-            <CardContent>
-              <Typography
-                variant="subtitle2"
-                color="textSecondary"
-                gutterBottom
-              >
-                <Grid container alignItems="center" spacing={1}>
-                  <Grid item>{labelCommand}</Grid>
-                  <Grid item>
-                    <Tooltip onClick={copyCommandLine} title={labelCopy}>
-                      <IconButton size="small">
-                        <IconCopyFile color="primary" fontSize="small" />
-                      </IconButton>
-                    </Tooltip>
-                  </Grid>
+        <Card>
+          <CardContent>
+            <Typography variant="subtitle2" color="textSecondary" gutterBottom>
+              <Grid container alignItems="center" spacing={1}>
+                <Grid item>{labelCommand}</Grid>
+                <Grid item>
+                  <Tooltip onClick={copyCommandLine} title={labelCopy}>
+                    <IconButton size="small">
+                      <IconCopyFile color="primary" fontSize="small" />
```

Comment:
We'll may need to remove the primary color. Cause I think the icon is too visible regarding important information of the panel

Reply:
But we'll wait feedbacks to manage this point